src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -5,7 +5,7 @@
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
     <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
     <AssemblyName>System.Private.ServiceModel</AssemblyName>
-    <AssemblyVersion>4.0.1.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <RootNamespace>System.ServiceModel</RootNamespace>
```

Comment:
I recommend we get sign-off on just this one change and get it merged asap.  I know the 3 stages are clearly articulated, but don't see any advantage to holding up step 1 for either steps 2 or 3.  We can do buddy-builds and TFS builds to ensure a smooth transition.
So step 1 (this file only) LGTM